python-regression/IXI/export-spent/index.js

```diff
@@ -0,0 +1,79 @@
+var System = java.lang.System;
+
+var spentAddressProvider = IOTA.spentAddressesProvider
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['IOTA' is not defined. (no-undef)](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=3955655)